pylabber/utils/utils.py

```diff
@@ -72,3 +74,28 @@ def get_prep_value(self, value):
         else:
             # Otherwise, just pass the value.
             return value
+
+
+class SubjectDataHandler:
+    @classmethod
+    def __init__(cls, path):
+        cls.questionnaire = QuestionnaireReader(path=path)
+
+    def open_password_protected_excel(
+        self, filename: str, password: str
+    ) -> io.BytesIO:
+        """
+        A method to open a password protected excel.
+
+        Returns
+        -------
+        io.BytesIO
+            Decrypted excel.
+        """
+
+        f = msoffcrypto.OfficeFile(open(filename, "rb"))
+        f.load_key(password=password)
+        decrypted = io.BytesIO()
+        f.decrypt(decrypted)
+
+        return decrypted
```

Comment:
Wouldn't this be better placed within the `QuestionnaireReader` class? I can't see why this additional class is required.

Reply:
Removed this method.